### DISCUSSION TITLE
How to solve the issue of background FP in yolov5s

### DISCUSSION THREAD

**Discussion by Ahmed-Sharba** [community/community#80888] [Programming Help]
### Select Topic Area

Question

### Body

Hi everyone I trained a dataset of two classes (vehicles and planes) with 2776 images including 249 background images in training, 781 images including 69 background images in validation, and 391 images including 32 background images in the test. The result is very good but the background in the confusion matrix is divided between two classes such as the picture uploaded. How can I prevent the FP of the background in the confusion matrix?
![confusion_matrix](https://github.com/community/community/assets/154312990/0cd43c51-a051-4470-8541-2a896cdc818f)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Ahmed-Sharba ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Reply by Ahmed-Sharba** [reply]
Thank you very much
